site/config.toml

```
@@ -57,6 +57,11 @@ disqusShortname = "checkly"
 	url = "/docs/browser-checks/"
 	weight = 30
 
+[[menu.resources]]
+	name = "Heartbeat checks"
+	url = "/docs/heartbeat-checks/"
+	weight = 30
```

Let's keep the weight a unique number.

```suggestion
	weight = 35
```